linux_os/guide/services/usbguard/usbguard_allow_hid/rule.yml

```diff
@@ -24,10 +24,6 @@ severity: medium
 identifiers:
     cce@rhel8: 82274-2
 
-references:
-    ospp: FMT_SMF_EXT.1
-    srg: SRG-OS-000114-GPOS-00059
```

Comment:
@dahaic why do you feel these references need to be removed? Seems like they should stay so that all of the reference is met.

Reply:
Allowing Class 03 USB devices is not a security feature. It is a convenience feature. And as such it does not seem right to be referenced as something that covers the requirement?

Reply:
(And the `usbguard_allow_*` rules are actually written as a convenience - OVALs there don't enforce anything, they pass when usbguard.rules file is non-empty)

Reply:
@dahaic does specify a Class 03 USB device "identify" a type of device?

Reply:
Also don't these rules assist in the configuring of USB interfaces i.e. allowing/disallowing what devices are recognized by the OS?

Reply:
Identification is done by the `usbguard` service - that running is enough to fulfill the requirements. Unfortunately default is too strict (nothing is allowed) so we need to sneak in something to soften it for general purpose system to be usable right after the installation :)

You are right that these rules are, in a way, related to the requirements . But they are optional extension. Should we still mark them with the references? These rules does not make system any more aligned with the requirement, and omission of the rule will make the system safer.

Keeping the references seems counter-intuitive to me.

Reply:
@redhatrises What do you think about @dahaic's explanation?

Reply:
The references still apply because organizationally I may allow class 3 devices whereas another organization may just want a strict policy that doesn't allow them. So the references need to be kept.

Reply:
@redhatrises So the use case for references is that they are used to identify rules related to a requirement that one may be interested to include in its policy?